_includes/head.html

```diff
@@ -6,6 +6,13 @@
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
 
+  <meta property="og:type" content="article">
+  <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
+  <meta property="og:description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
+  <meta property="og:url" content="{{ site.url }}{{ page.url }}">
+  <meta property="og:site_name" content="{{ site.title }}">
+  {% if page.image %}<meta property="og:image" content="{{ site.url }}/{{ page.image }}">{% endif %}
```

Comment:
Are all `page.image` relative paths?

Reply:
I randomly checked about 30 and they all seemed to be

Reply:
They look like this

```
image: img/bug_with_lens.png
```

However I just noticed there's a couple posts with another format so I'll change those too

Reply:
Updated the troublesome posts here: [`cbb031a` (#128)](https://github.com/MechanicalRock/mechanicalrock.github.io/pull/128/commits/cbb031a5d86e13174bbefac75add27453bce11a3)